x-pack/plugins/infra/common/inventory_models/types.ts

```diff
@@ -248,7 +248,6 @@ export const ESPercentileAggRT = rt.type({
 export const ESCaridnalityAggRT = rt.type({
   cardinality: rt.partial({
     field: rt.string,
-    script: rt.string,
```

Comment:
Not related to this change, but I was playing with `cardinality` aggregation to get the hosts count and found this attribute, which can't be used. https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-cardinality-aggregation.html#_script_4